src/platform.web.ts

```diff
@@ -3,7 +3,8 @@ export { d as debug };
 
 // === Export system-specific operations
 // Turn an AsyncIterable<Uint8Array> into a stream
-export { readableStreamFromIterable as itrToStream } from "https://deno.land/std@0.198.0/streams/readable_stream_from_iterable.ts";
+export const itrToStream = (itr: AsyncIterable<Uint8Array>) =>
+    ReadableStream.from(itr);
```

Comment:
Since this change, passing blob to `new InputFile()` lead to this error on **Vercel Edge Runtime**:

`TypeError: ReadableStream.from is not a function`

Example code:

```js
const response = await fetch(url, {headers, method, body});

const file = new InputFile(await response.blob(), filename);

await ctx.replyWithDocument(file);
```

![Снимок экрана 2023-10-07 в 19 12 07](https://github.com/grammyjs/grammY/assets/2877584/d55da6d1-8556-46e1-9a2d-49f8329453c7)